iot/api-client/manager/README.md

```diff
@@ -211,3 +211,56 @@ To publish state messages, run the sample as follows:
     gcloud beta pubsub subscriptions pull --auto-ack \
         projects/my-iot-project/subscriptions/my-subscription
 ```
+
+# Cloud IoT Core Java MQTT example
```

Comment:
Seems very long not to put an index at the top?

Reply:
Acknowledged, I'll have to clean up a few other things in the sample folder soon, I'll split up the READMEs when I do this.